src/app/jobs/weatherevents.job.ts

```diff
@@ -0,0 +1,53 @@
+// import { Mode } from '../../common/types';
+import { Job } from '../../common/job';
+import { IContainer } from '../../common/types';
+import { TextChannel } from 'discord.js';
+
+export class WeatherEventsJob extends Job {
+  // Run once an hour
+  public interval: number = 1000 * 60 * 60;
+
+  // For debugging purposes only
+  // public interval: number = 1000 * 10;
+  public name: string = 'weather_events';
+
+  constructor() {
+    super();
+  }
+
+  public async execute(container: IContainer) {
+    const channel = (container.guildService.getChannel("weather_events")) as TextChannel;
+    const resp = await container.httpService.get("https://api.weather.gov/alerts/active?area=FL");
+    const data: IWeather_Event = resp.data;
+
+    if (!data.features || data.features.length == 0) {
+      return;
+    }
+
+    const relevantData: IFeaturesEntity[] = data.features.filter(rD => {

```

Comment:
we don't really have a good reason for defining the `relevantData` variable name. i would suggest changing the variable name to `headlines` and chaining your `.map` call on line 34 to this chain of operations

also, for your filter function, naming it `rD` does not make sense. can you name it as `feature` ?